### PR TITLE
feat: elaborate on pull requests

### PR DESCRIPTION
**Pull request by fgheysels**

### CODE REVIEW

File: CONTRIBUTING.md
Comment (stijnmoreels) [inline CONTRIBUTING.md:21]: "nothing stops you" probably?

File: CONTRIBUTING.md
Comment (stijnmoreels) [inline CONTRIBUTING.md:29]: Code owners will review the pull requests.

File: CONTRIBUTING.md
Comment (stijnmoreels) [inline CONTRIBUTING.md:29]: Maybe the stuff on forking should be before the pull request stuff.

**Review by stijnmoreels — CHANGES_REQUESTED** [review]
Aha aha, great stuff. Happy that someone took the time to fix our sloppy spelling contribution guide 😉👍.

File: CONTRIBUTING.md
Comment (fgheysels) [reply CONTRIBUTING.md:21]: Yes, I was first going to write 'nothing stops you', but then decided to rephrase.  I'll modify this.

File: CONTRIBUTING.md
Comment (fgheysels) [reply CONTRIBUTING.md:29]: Yeah, I thought: anybody can review.

File: CONTRIBUTING.md
Comment (fgheysels) [reply CONTRIBUTING.md:29]: Difficult :)  In the PR part, we're indeed referring to forking.   How it must be done is then explained later.
If you turn it around, you indeed explain what forking is, but then one might wonder:  why do I need to know this.  And the aha-erlebnis comes later.   I think both options are valid.

File: CONTRIBUTING.md
Comment (stijnmoreels) [inline CONTRIBUTING.md:21]: This is an improvement, Thanks!

File: CONTRIBUTING.md
Comment (stijnmoreels) [reply CONTRIBUTING.md:29]: Yes, and maybe we should support that, but I'm not sure if we should put that in our contribution guide. Like this, it seems that any contributor could approve and merge a pull request.

File: CONTRIBUTING.md
Comment (stijnmoreels) [reply CONTRIBUTING.md:29]: Yes, you're right! In this case, I would make forking a sub-title of pull requests so it's more clear how they belong together.

**Comment by stijnmoreels** [comment]
Feel free to check my suggestions, @fgheysels 😉

File: CONTRIBUTING.md
Comment (fgheysels) [reply CONTRIBUTING.md:29]: Actually, only code-owners have the rights to merge a pull-request.   In theory, everyone can review, but it's code-owners that have the final word.   I guess that is expressed in that text ?

File: CONTRIBUTING.md
Comment (fgheysels) [reply CONTRIBUTING.md:29]: I'll do that; good idea for adding it as a sub

File: CONTRIBUTING.md
Comment (stijnmoreels) [reply CONTRIBUTING.md:29]: Aha, ok, I didn't saw that. Yes, that will be ok. Thx.

**Comment by stijnmoreels** [comment]
Problem with branch, we'll create follow-up PR.

**Pull request merged**
